ui/src/components/header/UserMenu.vue

```diff
@@ -17,6 +17,132 @@
+                    </h3>
+                    <small>{{ $t('label.action.create.volume') }}</small>
+                  </a-col>
+                </a-row>
+              </router-link>
+            </a-menu-item>
+            <a-menu-item style="width: 100%; padding: 12px">
+              <router-link :to="{ path: '/guestnetwork', query: { action: 'createNetwork' } }">
+                <a-row>
+                  <a-col style="margin-right: 12px">
+                    <a-avatar :style="{ backgroundColor: $config.theme['@primary-color'] }">
+                      <template #icon>
+                        <apartment-outlined />
+                      </template>
+                    </a-avatar>
+                  </a-col>
+                  <a-col>
+                    <h3 style="margin-bottom: 0px;">
+                      {{ $t('label.network') }}
+                    </h3>
+                    <small>{{ $t('label.add.network') }}</small>
+                  </a-col>
+                </a-row>
+              </router-link>
+            </a-menu-item>
+            <a-menu-item style="width: 100%; padding: 12px">
+              <router-link :to="{ path: '/vpc', query: { action: 'createVPC' } }">
+                <a-row>
+                  <a-col style="margin-right: 12px">
+                    <a-avatar :style="{ backgroundColor: $config.theme['@primary-color'] }">
+                      <template #icon>
+                        <deployment-unit-outlined />
+                      </template>
+                    </a-avatar>
+                  </a-col>
+                  <a-col>
+                    <h3 style="margin-bottom: 0px;">
+                      {{ $t('label.vpc') }}
+                    </h3>
+                    <small>{{ $t('label.add.vpc') }}</small>
+                  </a-col>
+                </a-row>
+              </router-link>
+            </a-menu-item>
+            <a-menu-item style="width: 100%; padding: 12px">
+              <router-link :to="{ path: '/template', query: { action: 'registerTemplate' } }">
+                <a-row>
+                  <a-col style="margin-right: 12px">
+                    <a-avatar :style="{ backgroundColor: $config.theme['@primary-color'] }">
+                      <template #icon>
+                        <picture-outlined />
+                      </template>
+                    </a-avatar>
+                  </a-col>
+                  <a-col>
+                    <h3 style="margin-bottom: 0px;">
+                      {{ $t('label.templatename') }}
+                    </h3>
+                    <small>{{ $t('label.action.register.template') }}</small>
+                  </a-col>
+                </a-row>
+              </router-link>
+            </a-menu-item>
+          </a-menu>
+        </template>
+        <a-button type="primary">
+          {{ $t('label.create') }}
+          <DownOutlined />
+        </a-button>
+      </a-dropdown>
+    </span>
```

Comment:
This code block can be extracted to a component.

Reply:
Thanks for reviewing @GutoVeronezi I'll explore where it can be refactored, there are a couple of places on dashboard too where cards can be re-usable components. In this case, the kubernetes icon is not consistent with others though (uses fa-icon than ant design's icon).

Reply:
Done @GutoVeronezi refactored these in a separate CreateMenu component.